docs/interactions/Slash_Commands.md

```diff
@@ -682,11 +684,11 @@ Create a new global command. New global commands will be available in all guilds
 
 ###### JSON Params
 
-| Field       | Type                                                                                            | Description                                  |
-|-------------|-------------------------------------------------------------------------------------------------|----------------------------------------------|
-| name        | string                                                                                          | 1-32 character name matching `^[\w-]{1,32}$` |
-| description | string                                                                                          | 1-100 character description                  |
-| options?    | array of [ApplicationCommandOption](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommandoption) | the parameters for the command               |
+| Field       | Type                                                                                            | Description                                      |
+|-------------|-------------------------------------------------------------------------------------------------|--------------------------------------------------|
+| name        | string                                                                                          | 1-32 character name matching `^[a-z0-9-]{1,32}$` |
```

Comment:
This is actually incorrect since the name supports more than just a-z. Better would be `\p{Ll}\p{N}` for lowercase letters and numbers. Or maybe just doing this:
```suggestion
| name        | string                                                                                          | 1-32 character lowercase name matching `^[\w-]{1,32}$` |
```

Reply:
That makes sense - hence my comment w.r.t. non-Latin characters. My concern with _saying_ lowercase but not providing the regexp for it is that users tend to rely on the regexp, and so making it "copy paste" is likely a slightly better dev. experience.

Since `\w` is typically equivalent to `[A-Za-z0-9_]` in many regexp engines, `^[\p{Ll}\p{N}_-]{1,32}$` is likely the right replacement here.

Thoughts?

Reply:
I feel like just stating lowercase instead would be better than using some exotic regex character classes.

Reply:
> ^[\p{Ll}\p{N}_-]{1,32}$
If we're still going the regex route, would `^[a-z0-9_]{1,32}$` not be easier to read? doesn't that have the same effect?